test/onigumo_test.exs

```diff
@@ -111,4 +105,12 @@ defmodule OnigumoTest do
   defp body(url) do
     "Body from: #{url}\n"
   end
+
+  defp assert_downloaded(url, tmp_dir) do
+    file_name = Base.url_encode64(url, padding: false)
```

Comment:
This is going to be replaced with `Onigumo.create_filename` once #82 is merged. This function has its own test and as such it is trusted.